hack/bazooka-compose/docker-compose.yml

```diff
@@ -1,29 +1,50 @@
-server:
-  container_name: bzk_server
-  image: bazooka/server
-  ports:
-  - "3000:3000"
-  - "3001:3001"
-  volumes:
-  - "${BZK_HOME}:/bazooka"
-  - "/var/run/docker.sock:/var/run/docker.sock"
-  environment:
-  - "BZK_DOCKERSOCK=/var/run/docker.sock"
-  - "BZK_API_URL=http://bzk_server:3000"
-  - "BZK_SYSLOG_URL=tcp://${BZK_HOST}:3001"
-  - "BZK_DB_URL=bzk_db:27017"
-  - "BZK_NETWORK=bazookacompose"
-  - "BZK_SCM_KEYFILE=${BZK_SCM_KEYFILE}"
-  - "BZK_HOME=${BZK_HOME}"
+version: '2'
 
-web:
-  container_name: bzk_web
-  image: bazooka/web
-  ports:
-  - "8000:80"
-  environment:
-  - "BZK_SERVER_HOST=bzk_server"
+services:
+  server:
+    container_name: bzk_server
+    image: bazooka/server
+    ports:
+    - "3000:3000"
+    - "3001:3001"
+    volumes:
+    - "${BZK_HOME}:/bazooka"
+    - "/var/run/docker.sock:/var/run/docker.sock"
+    environment:
+    - "BZK_DOCKERSOCK=/var/run/docker.sock"
+    - "BZK_API_URL=http://bzk_server:3000"
+    - "BZK_SYSLOG_URL=tcp://${BZK_HOST}:3001"
+    - "BZK_DB_URL=bzk_db:27017"
+    - "BZK_NETWORK=bazookacompose"
```

Comment:
BTW, compose still prefixes the network name with the folder name, so in this case the env var value should be `bazookacompose_bazooka`.